.github/CODEOWNERS

```
@@ -0,0 +1,7 @@
+# CODEOWNERS: https://help.github.com/articles/about-codeowners/
+
+# Docs 
+*.md @citmc @cmwaters @nooomski @coldice 
```

```suggestion
*.md @citmc @cmwaters @nooomski @xavierlepretre @coldice 
```
We would like to add Xavier here